VPEmitter _ VPEmitter 추가

sanghun219 opened the pull request:
# 세부사항
1. VPEmitter 파라미터 편집
 - Emitter Actor를 생성하고 ParticleSystem 컴포넌트를 추가해 각종 파라미터 값들( 색상, 입자 개수, 입자 크기, 시작 크기, 딜레이, 알파값, 생명주기)을 조절하도록 함.
 - 사용자들이 파티클 파라미터를 조절하기 위해서는 파티클의 Ditribution 값을 Particle Param으로 변경하고 ParamMode를 DPM Direct로 바꾸고 파라미터가 가질수있는 최대값으로 설정해두어야한다.

bangbangdog219 committed:
VPCameraActor _ VPCameraActor 추가

# 세부사항
1.  Actor에 카메라 생성 및 조작
  - Actor에대해 고정, 상대적인 Transform을 가지도록 생성, FieldofView나 카메라의 위치 회전을 설정가능.
2. Actor의 외부적 요인
  - Actor당 붙을 수 있는 최대 카메라 수, 2개 이상의 카메라에 대한 카메라 전환 기능, 기즈모나 UI적 요소에 의한
    Transform 및 카메라 설정에 대한 것들은 외부 Class에서 조작.

bangbangdog219 committed:
VPCameraActor _ VPCameraActor 추가

#세부사항
 1. Actor에 카메라 생성 및 조작
   - Actor에대해 고정, 상대적인 Transform을 가지도록 생성, FieldofView나 카메라의 위치 회전을 설정가능.
  2. Actor의 외부적 요인
    - Actor당 붙을 수 있는 최대 카메라 수, 2개 이상의 카메라에 대한 카메라 전환 기능, 기즈모나 UI적 요소에 의한
Transform 및 카메라 설정에 대한 것들은 외부 Class에서 조작.

Minusi committed:
Merge branch 'master' into Interaction

bangbangdog219 committed:
VPEmitter _ VPEmitter 추가

# 세부사항
1. VPEmitter 파라미터 편집
 - Emitter Actor를 생성하고 ParticleSystem 컴포넌트를 추가해 각종 파라미터 값들( 색상, 입자 개수, 입자 크기, 시작 크기, 딜레이, 알파값, 생명주기)을 조절하도록 함.
 - 사용자들이 파티클 파라미터를 조절하기 위해서는 파티클의 Ditribution 값을 Particle Param으로 변경하고 ParamMode를 DPM Direct로 바꾸고 파라미터가 가질수있는 최대값으로 설정해두어야한다.

Minusi commented on UEPrototype/Source/UEPrototype/Public/ActorInfo/VPEmitter.h:
```diff
@@ -0,0 +1,65 @@
+	
+	
+private:
+	UPROPERTY()
```
Cashed(x) -> Cached(o)

Minusi commented on UEPrototype/Source/UEPrototype/Public/ActorInfo/VPCameraActor.h:
```diff
@@ -0,0 +1,148 @@
+	UFUNCTION(BlueprintCallable, Category = "VPCameraActor")
+	APlayerCameraManager* GetPlayerCameraMgr() const;
+
+	// 플레어이 카메라에 대한 레퍼런스
```
플레이어

bangbangdog219 committed:
VPEmitter _ VPEmitter 추가

# 세부사항 _ 수정
1. VPEmitter 파라미터 편집
 - Emitter Actor를 생성하고 ParticleSystem 컴포넌트를 추가해 각종 파라미터 값들( 색상, 입자 개수, 입자 크기, 시작 크기, 딜레이, 알파값, 생명주기)을 조절하도록 함.
 - 사용자들이 파티클 파라미터를 조절하기 위해서는 파티클의 Ditribution 값을 Particle Param으로 변경하고 ParamMode를 DPM Direct로 바꾸고 파라미터가 가질수있는 최대값으로 설정해두어야한다.

Minusi committed:
Merge branch 'prototype' into Interaction
